templates/testpmd/resource-specs/sriov-vm1-template.yaml

```
@@ -0,0 +1,112 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: dpdk-vm-template
+objects:
+- apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    annotations:
+      openshift.io/display-name: "RHEL VM with SR-IOV-backed NIC and DPDK."
+      description: >-
+        Template for RHEL 8.4 VM, with a secondary interface backed by an SR-IOV node NIC.
+        Involves replacing the default NIC driver (iavf) to vfio-pci, to enable running DPDK.
+      openshift.io/documentation-url: "https://github.com/RHsyseng/cnv-supplemental-templates/tree/main/templates/testpmd"
+    name: ${VM_NAME}
```

```suggestion
    name: ${NAME}
```
revisited the templates - to keep it aligned with other templates we provide, please rename.

---

Ack